integration/benchmark/contLoadGenerator.go

```diff
@@ -157,7 +158,19 @@ func New(
 		availableAccountsLo: loadParams.NumberOfAccounts,
 	}
 
-	return lGen, nil
+	// TODO(rbtz): hide load implementation behind an interface
+	switch loadParams.LoadType {
+	case TokenTransferLoadType:
+		lg.workFunc = lg.sendTokenTransferTx
+	case TokenAddKeysLoadType:
+		lg.workFunc = lg.sendAddKeyTx
+	case ConstExecCostLoadType:
+		lg.workFunc = lg.sendConstExecCostTx
+	default:
+		lg.workFunc = lg.sendFavContractTx
```

Comment:
better to be more strict:
```suggestion
	case FavContractLoadType: // Leo: I'm guessing the type here, please use the actual type
		lg.workFunc = lg.sendFavContractTx
	default:
		return nil, fmt.Errorf("unknown load type: %v", loadParams.LoadType)	
```